cmake/Modules/FindGMP.cmake

```diff
@@ -19,9 +19,9 @@ endif()
 
 if(NOT USE_SYSTEM_GMP)
 	message(STATUS "Using bundled mini-gmp library.")
-	set(GMP_INCLUDE_DIR ${PROJECT_SOURCE_DIR}/gmp)
+	set(GMP_INCLUDE_DIR ${PROJECT_SOURCE_DIR}/../lib/gmp)
 	set(GMP_LIBRARY gmp)
-	add_subdirectory(gmp)
+	add_subdirectory(${PROJECT_SOURCE_DIR}/../lib/gmp ${PROJECT_BINARY_DIR}/../lib/gmp)
```

Comment:
Why are you adding two paths here?

If the reason is that it doesn't work otherwise, then: no, sorry this is way too hacky.